stan/math/rev/mat/fun/divide.hpp

```diff
@@ -9,18 +9,165 @@
 namespace stan {
 namespace math {
 
+namespace internal {
+template <int R, int C>
+class matrix_scalar_divide_vari_dv : public vari {
+ public:
+  int rows_;
+  int cols_;
+  vari* adjCRef_;
+  vari** adjResultRef_;
+  double invc;
+
+  explicit matrix_scalar_divide_vari_dv(const Eigen::Matrix<double, R, C>& m,
+                                        const var& c)
+      : vari(0.0),
```

Comment:
[optional]
`vari(0)` is fine---we don't need the `.0` everywhere. The only place it should be necessary is when you don't want integer rounding when dividing two integers.  If it's necessary elsewhere, that's a bug that should be reported.

Reply:
vari(0) it is then; thanks.